modules/ztp-configuring-ptp-fast-events.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-configuring-ptp-fast-events_{context}"]
+= Configuring PTP fast events using PolicyGenTemplate custom resources and argoCD
```

Comment:
Maybe:
Configuring PTP fast events using PolicyGenTemplate custom resources and GitOps ZTP